PhysicsTools/PatAlgos/python/tools/jetTools.py

```diff
@@ -536,6 +536,21 @@ def setupBTagging(process, jetSource, pfCandidates, explicitJTA, pvSource, svSou
                 addToProcessAndTask(btagPrefix+btagInfo+labelName+postfix,
                                     btag.softPFElectronsTagInfos.clone(jets = jetSource, primaryVertex=pvSource, electrons=elSource),
                                     process, task)
+
+            if btagInfo == 'pfDeepFlavourTagInfos':
+                # as if now the training tag info is based on corrected jets
+                # so we expect a bit of a loss on performance
```

Comment:
does this mean that the analysts will have to rerun the tagger in some other way (on miniAOD output?)  to get the right "approved" performance on corrected jets?
If so, the implementation will need to change so that we write the correct tag infos by default.

Reply:
this is a legacy comment, the model was retrained in uncorrected jets by @jkiesele, will remove the comment

Reply:
removed comment in https://github.com/cms-sw/cmssw/pull/19893/commits/a3cab4ef3f5171f01a7b710aa1b2e9686b31e8cf